adal/src/main/java/com/microsoft/aad/adal/Oauth2.java

```diff
@@ -303,6 +303,36 @@ public String buildRefreshTokenRequestMessage(String refreshToken)
                 AuthenticationConstants.OAuth2.CLIENT_INFO,
                 AuthenticationConstants.OAuth2.CLIENT_INFO_TRUE
         );
+        
+        message = buildRequestMessage(message);
+        return message;
+    }
+
+    public String buildAssertionMessage(@NonNull String assertion,@NonNull String assertionType)
+            throws UnsupportedEncodingException {
+        Logger.v(TAG, "Building request message for redeeming token with saml assertion.");
+        String message = String.format("%s=%s&%s=%s&%s=%s&%s=%s&%s=%s",
+                AuthenticationConstants.OAuth2.GRANT_TYPE,
+                StringExtensions.urlFormEncode(assertionType),
+
+                AuthenticationConstants.OAuth2.ASSERTION,
```

Comment:
Looks like constant is still being imported from common, let's fix this as we moved the constant to ADAL